parsl/executors/high_throughput/zmq_pipes.py

```diff
@@ -37,8 +55,21 @@ def run(self, message):
         in ZMQ sockets reaching a broken state once there are ~10k tasks in flight.
         This issue can be magnified if each the serialized buffer itself is larger.
         """
-        self.zmq_socket.send_pyobj(message, copy=True)
-        reply = self.zmq_socket.recv_pyobj()
+        try:
+            self.zmq_socket.send_pyobj(message, copy=True)
+            reply = self.zmq_socket.recv_pyobj()
+        except zmq.ZMQError:
+            logger.error("Potential ZMQ REQ-REP deadlock caught")
```

Comment:
is there any more interesting info stored in the ZMQError (now or in future) that would be useful for debugging this? if so, log at least the message from it?

Reply:
Should've used logger.exception instead that would capture this info. Will switch.